dev/CommonStyles/ToggleSwitch_themeresources_v1.xaml

```diff
@@ -188,9 +188,9 @@
          </ResourceDictionary>
     </ResourceDictionary.ThemeDictionaries>
 
-    <Thickness x:Key="ToggleSwitchTopHeaderMargin">0,0,0,0</Thickness>
-    <x:Double x:Key="ToggleSwitchPreContentMargin">6</x:Double>
-    <x:Double x:Key="ToggleSwitchPostContentMargin">6</x:Double>
+    <Thickness x:Key="ToggleSwitchTopHeaderMargin">0,0,0,4</Thickness>
```

Comment:
Marcel, we can leave the v1 styles where they are, so please undo this.

Reply:
Reverted now @ranjeshj .